dao/tx/mempool_tx.go

```diff
@@ -27,56 +27,27 @@ const (
 	MempoolTableName = `mempool_tx`
 )
 
-const (
-	PendingTxStatus = iota
-	ExecutedTxStatus
-	SuccessTxStatus
-	FailTxStatus
-)
-
 type (
 	MempoolModel interface {
```

Comment:
can we change the name of the MempoolModel

Reply:
How about TxPoolModel?